docs/getting-started/quickstart.js-backend.mdx

```diff
@@ -443,85 +475,37 @@ The [`Auth`](/docs/reference/backend/types/auth-object) object contains importan
-            }
-
-            const user = await createClerkClient({ secretKey: process.env.CLERK_SECRET_KEY }).users.getUser(
-              userId,
-            )
-
-            // Return the user
-            return { serialisedUser: JSON.stringify(user) }
-          }
-          ```
-        </CodeBlockTabs>
-      </Tab>
-
-      <Tab>
-        For Tanstack React Start, the `Auth` object is accessed using the `getAuth()` function. [Learn more about using `getAuth()`](/docs/tanstack-react-start/guides/users/reading#server-side).
+        For Tanstack React Start, the `Auth` object is accessed using the `auth()` function. [Learn more about using `auth()`](/docs/tanstack-react-start/guides/users/reading#server-side).
 
         ```tsx {{ filename: 'app/routes/api/example.tsx' }}
-        import { createClerkClient } from '@clerk/backend'
         import { json } from '@tanstack/react-start'
-        import { createAPIFileRoute } from '@tanstack/react-start/api'
-        import { getAuth } from '@clerk/tanstack-react-start/server'
-
-        export const Route = createAPIFileRoute('/api/example')({
-          GET: async ({ request, params }) => {
-            // Use `getAuth()` to access `isAuthenticated` and the user's ID
-            const { isAuthenticated, userId } = await getAuth(req)
-
-            // Protect the route by checking if the user is signed in
-            if (!isAuthenticated) {
-              return json({ error: 'Unauthorized' }, { status: 401 })
-            }
-
-            // Instantiate the JS Backend SDK
-            const clerkClient = createClerkClient({ secretKey: import.meta.env.CLERK_SECRET_KEY })
-
-            // Use the JS Backend SDK's `getUser()` method to get the Backend User object
-            const user = await clerkClient.users.getUser(userId)
-
-            // Return the Backend User object
-            return json({ user })
+        import { createFileRoute } from '@tanstack/react-router'
+        import { auth, clerkClient } from '@clerk/tanstack-react-start/server'
+
+        export const ServerRoute = createFileRoute('/api/example')({
+          server: {
+            handlers: {
+              GET: async () => {
+                // The `Auth` object gives you access to properties like `isAuthenticated` and `userId`
+                // Accessing the `Auth` object differs depending on the SDK you're using
+                // https://clerk.com/docs/reference/backend/types/auth-object#how-to-access-the-auth-object
+                const { isAuthenticated, userId } = await auth()
+
+                // Protect the route by checking if the user is signed in
+                if (!isAuthenticated) {
+                  return json({ error: 'Unauthorized' }, { status: 401 })
+                }
+
+                // Initialize the JS Backend SDK
+                // This varies depending on the SDK you're using
+                // https://clerk.com/docs/js-backend/getting-started/quickstart
```

Comment:
I like the helpfulness of adding a link here, but they are going to go unchecked if/when we do updates or redirects. 

Because of that, should we leave them or remove them?